_includes/v21.1/orchestration/kubernetes-remove-nodes-helm.md

```diff
@@ -0,0 +1,127 @@
+
+{{site.data.alerts.callout_danger}}
+If you remove nodes without first telling CockroachDB to decommission them, you may cause data or even cluster unavailability. For more details about how this works and what to consider before removing nodes, see [Decommission Nodes](remove-nodes.html).
+{{site.data.alerts.end}}
+
+1. Use the [`cockroach node status`](cockroach-node.html) command to get the internal IDs of nodes. For example, if you followed the steps in [Deploy CockroachDB with Kubernetes](deploy-cockroachdb-with-kubernetes.html#step-3-use-the-built-in-sql-client) to launch a secure client pod, get a shell into the `cockroachdb-client-secure` pod:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl exec -it cockroachdb-client-secure \
+    -- ./cockroach node status \
+    --certs-dir=/cockroach-certs \
+    --host=my-release-cockroachdb-public
+    ~~~    
+
+    ~~~
+      id |                                     address                                     | build  |            started_at            |            updated_at            | is_available | is_live
+    +----+---------------------------------------------------------------------------------+--------+----------------------------------+----------------------------------+--------------+---------+
+       1 | my-release-cockroachdb-0.my-release-cockroachdb.default.svc.cluster.local:26257 | {{page.release_info.version}} | 2018-11-29 16:04:36.486082+00:00 | 2018-11-29 18:24:24.587454+00:00 | true         | true
+       2 | my-release-cockroachdb-2.my-release-cockroachdb.default.svc.cluster.local:26257 | {{page.release_info.version}} | 2018-11-29 16:55:03.880406+00:00 | 2018-11-29 18:24:23.469302+00:00 | true         | true
+       3 | my-release-cockroachdb-1.my-release-cockroachdb.default.svc.cluster.local:26257 | {{page.release_info.version}} | 2018-11-29 16:04:41.383588+00:00 | 2018-11-29 18:24:25.030175+00:00 | true         | true
+       4 | my-release-cockroachdb-3.my-release-cockroachdb.default.svc.cluster.local:26257 | {{page.release_info.version}} | 2018-11-29 17:31:19.990784+00:00 | 2018-11-29 18:24:26.041686+00:00 | true         | true
+    (4 rows)
+    ~~~
+
+    The pod uses the `root` client certificate created earlier to initialize the cluster, so there's no CSR approval required.
+
+1. Use the [`cockroach node decommission`](cockroach-node.html) command to decommission the node with the highest number in its address, specifying its ID (in this example, `4`):
+
+    {{site.data.alerts.callout_info}}
+    It's important to decommission the node with the highest number in its address because, when you reduce the replica count, Kubernetes will remove the pod for that node.
+    {{site.data.alerts.end}}
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl exec -it cockroachdb-client-secure \
+    -- ./cockroach node decommission 4 \
```

Comment:
@keith-mcclellan this didn't work for the operator, does this need to be updated?

Reply:
the helm chart is still co-presenting GRPC and SQL on 26257, so this command will still work there.